packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -97,14 +102,38 @@ export const ParticipantItem = (props: ParticipantItemProps): JSX.Element => {
     />
   );
 
-  const meTextStyle = mergeStyles({ color: theme.palette.neutralTertiary }, styles?.me);
-  const contextualMenuStyle = mergeStyles({ background: theme.palette.neutralLighterAlt }, styles?.menu);
-  const menuButtonContainerStyle = mergeStyles({
-    root: {
-      '&:hover': { background: 'none' },
-      '&:active': { background: 'none' }
-    }
-  });
+  const meTextStyle = useMemo(
```

Comment:
Here and below: You don't need to memoize the result of mergeStyles as it's a string (CSS class name).

Reply:
Sorry I may have missed something but I thought useMemo is for variables that are computation heavy that only depend on a few props. I'm trying to avoid using mergeStyles if theme and styles have not changed. For example, when a participant unmutes, I dont want to call mergeStyles again.

Reply:
Oh, OK.

`useMemo` is used more to avoid creating a new result object, to avoid rerendering a Component that uses that object as a prop. It can be used to simply avoid computation as you do here as well, but usually computations like this aren't the costly step.